common/lib/xmodule/xmodule/css/problem/edit.scss

```diff
@@ -23,39 +23,22 @@
       }
     }
 
-    .cheatsheet-toggle {
-      width: 21px;
-      height: 21px;
-      padding: 0;
-      margin: -1px 5px 0 15px;
-      border-radius: 22px;
-      border: 1px solid #a5aaaf;
-      background: #e5ecf3;
-      font-size: 13px;
-      font-weight: 700;
-      color: #565d64;
-      text-align: center;
-    }
   }
 }
 
 .simple-editor-cheatsheet {
   position: absolute;
-  top: 0;
-  left: 100%;
+  top: 41px;
+  left: 85%;
   width: 0;
-  border-radius: 0 3px 3px 0;
-
-  @include linear-gradient(left, $shadow-l1, $transparent 4px);
+  border-left: 1px solid $gray-l2;
 
-  background-color: $white;
+  background-color: $lightGrey;
   overflow: hidden;
 
-  @include transition(width 0.3s linear 0s);
-
   &.shown {
-    width: 20%;
-    height: 100%;
+    width: 15%;
```

Comment:
Change #2 - can shift to 30%

Reply:
on .cheatsheet-wrapper below, where padding is 10%, as part of Change #2 - can shift to 5%